cmd/fpga_plugin/fpga_plugin.go

```diff
@@ -376,10 +365,10 @@ func main() {
 
 	plugin, err := newDevicePlugin(mode)
 	if err != nil {
-		fatal(err)
+		klog.Fatal(err)
```

Comment:
And here would be useful if the program produced a stack trace that our users could just copy&paste in their bug reports.

Reply:
changed to a `(%+v, err)`? Note also, `klog.Fatal` [prints out a stack dump of all running goroutines](https://github.com/kubernetes/klog/blob/master/klog.go#L1316-L1321), which is not quite the same as the stack of where the error came from, but similar-ish...

Reply:
Yes, but for the current goroutine it prints the trace to the line where `Fatal()` was called, not the origin of the actual error.